packages/qdrant-loader-core/src/qdrant_loader_core/graph/falkor_store.py

```
@@ -0,0 +1,370 @@
+            LIMIT {MAX_ROWS}
+            """
+        result = await asyncio.to_thread(
+            self._graph.query,
+            query,
+            params,
+        )
+        nodes_map: dict[str, GraphNode] = {}
+        internal_node_id_map: dict[int, str] = {}
+        edges: list[GraphEdge] = []
+
+        def _extract_node(node_value):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_extract_node`

(ANN202)